FLOW_EXAMPLE.md

```diff
@@ -0,0 +1,61 @@
+Flow example of Bors-NG
+=======================
+
+This document goal is to explain what's the flow of events which occur when
+someone merges a PR with bors.
+
+PR creation
+-----------
+Nothing special in this step. A developer creates a PR, CI runs and other developers
+do the code review. After everyone is in agreement, someone writes `bors r+`
+as a comment in the PR.
+
+Comments web hook
+-----------------
+Every time there is a comment, bors gets notified through a Github webhook.
+See `webhoook_controller.ex` for the code.
+Bors gets all the information it needs and creates a `Command`.
+If bors parses a command, it starts a new thread to do the work.
```

Comment:
It does not start a new thread. What it does is look up, in the registry, if there's an existing Batcher thread.

If the batcher thread does not exist (either because there was no running work, or because it crashed), it creates a new thread.

If the batcher thread does already exist, it just sends a message to it.